packages/widgetbook/lib/src/widgets/expanders/expand_button.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/providers/organizer_provider.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+class ExpandButton extends StatefulWidget {
+  const ExpandButton({
+    Key? key,
+    required this.organizers,
+    required this.expandTo,
+    this.size = 17,
+  }) : super(key: key);
```

Comment:
as i already stated: `size = 17` is a bit small (tho looks good). Material Design recommends a size of at least `48 times 48` (see: https://material.io/develop/web/supporting/touch-target) but in this case i think its fine if its smaller than 48 because users will be on a desktop device which allows for more precise targeting of UI elements

Reply:
Oh I didn't know about that, I'll change it to 32 then 👍.